app/components/typography/description.tsx

```diff
@@ -14,6 +14,7 @@ const Description: FC<TextProps> = ({ children, style }) => (
       },
       style
     )}
+    {...rest}
```

Comment:
Tady se mi nepropisovala definice toho omezeni description na dva radky. Omlouvam se screenshot kde to fungovalo, nevim co bylo spatne 🙈